### ISSUE TITLE
list installed returns nothing on Paper

### ISSUE THREAD

**Issue by SlimeDog** [PlaceholderAPI/PlaceholderAPI#534]
### Issue

On Paper 1.16.4, `papi ecloud list installed` returns nothing. On Spigot 1.16.4, with the same expansion set, it returns the expected list. Except for running Paper vs Spigot, the servers are identical -- one is a clone of the other.

### Expected behaviour

Return the same list on both platforms.

### Actual behaviour

Returns nothing.

### How to reproduce

- Install PAPI 2.10.10-DEV-122 (and probably previous versions)
- Run the following commands
  - papi ecloud enable
  - papi ecloud list installed
  - papi ecloud disable

### `/papi dump` output

Will provide on request, if necessary. Otherwise, I prefer not to publish my plugin list.

**Comment by Andre601** [comment]
Can't reproduce using latest PAPI dev build (122)
![image](https://user-images.githubusercontent.com/11576465/104135769-4b625c00-5392-11eb-83a9-9c730a7b0f75.png)

**Comment by SlimeDog** [comment]
Paper 1.16.4 build 401
PlaceholderAPI 2.10.10-DEV-122

**Comment by Andre601** [comment]
Could you perhaps provide the list of expansions (Including those registered by plugins)

**Comment by Andre601** [comment]
I also still can't reproduce with latest PaperMC build
![image](https://user-images.githubusercontent.com/11576465/104135961-6a152280-5393-11eb-8494-a34c2b4c9105.png)

**Comment by SlimeDog** [comment]
On Spigot only
```
>papi ecloud list installed
[13:03:12] [Server thread/INFO]: Installed Expansions
[13:03:12] [Server thread/INFO]:      Name            Author   Verified  Version  
[13:03:12] [Server thread/INFO]: 1.   CoordinateTool  vk2gpz   Y         2.1.0    
[13:03:12] [Server thread/INFO]: 2.   Disease         artitus  Y         0.0.2    
[13:03:12] [Server thread/INFO]: 3.   Essentials      clip     Y         1.2.4    
[13:03:12] [Server thread/INFO]: 4.   LuckPerms       luck     Y         5.0-R2   
[13:03:12] [Server thread/INFO]: 5.   Multiverse      clip     Y         1.0.1    
[13:03:12] [Server thread/INFO]: 6.   Player          clip     Y         1.7.2    
[13:03:12] [Server thread/INFO]: 7.   Quests          pikamug  Y         1.4      
[13:03:12] [Server thread/INFO]: 8.   Server          clip     Y         2.4.0    
[13:03:12] [Server thread/INFO]: 9.   Statistic       clip     Y         2.0.0    
[13:03:12] [Server thread/INFO]: 10.  Thirst          cj89898  Y         1.2.1    
[13:03:12] [Server thread/INFO]: 11.  Vault           clip     Y         1.7.0    
[13:03:12] [Server thread/INFO]: 12.  ViaVersion      clip     Y         1.0.0    
[13:03:12] [Server thread/INFO]: 13.  WorldGuard      clip     Y         1.4.2  
```
On Paper, empty.

On Spigot or Paper
```
>papi list
[13:27:23 INFO]: A total of 30 placeholder hook(s) are active: 
bentobox, biomeremap, bskyblock, challenges, combatlogx, coordinatetool, deluxemenus, denizen, disease, essentials
frp, insights, level, luckperms, mobhunting, multiverse, ocm, plan, player, pyroweatherpro
quests, residence, server, spark, stat, statistic, thirst, vault, viaversion, worldguard
```

**Comment by Andre601** [comment]
So the command does return just nothing? Not even a mention that there aren't any expansions installed?

Sounds more like a plugin may hide this message or mess up the console somehow, as the code I checked really wouldn't give any reason to return nothing.

**Comment by SlimeDog** [comment]
The complete transaction on my Paper server is
```
> papi ecloud enable
> papi ecloud list installed
> papi ecloud disable
```
That is, no output at all.

**Comment by Andre601** [comment]
Can you try this on a fresh server and with just the expansions from the expansions folder (from the ecloud)?
If it lists the expansions there then it's pretty much caused by a plugin acting weird on Paper and hiding/messing up PAPI messages.

**Comment by SlimeDog** [comment]
I will run some isolations.

**Comment by SlimeDog** [comment]
It's a ConsoleSpamFix configuration issue. Paper treats some nearly-empty lines differently from Spigot. :(

Apologies for the churn.

**Comment by Andre601** [comment]
No worries.

I for my part don't use the ConsoleSpamFix plugin as doing so could cause issues like this one, or hide usually important (error) messages.

**Comment by SlimeDog** [comment]
> ... as doing so could cause issues like this one, or hide usually important (error) messages.

I am very careful about what I hide, but this one slipped by. It turns out to be the long line of hyphens, which obscures the entire `papi ecloud list` output, since it was changed to one (long) JSON string a few months ago, rather than separate lines. For some reason, it is not elided in Spigot, which caused the confusion when we started running a Paper server in parallel.
